Server/src/services/DataUpload/UnapprovedUploadService.ts

```diff
@@ -67,7 +67,7 @@ export class UnapprovedUploadService extends AbstractUploadService {
 
         await this.uploadModel.createEntryInUnapprovedDataSets(dataSetID)
 
-        requestResponse.message = "Upload to Database was successful!"
+        requestResponse.message = dataSetID.toString()
```

Comment:
We changed this to return the datasetId? Why?
This is the api call after a user uploads to DB

Reply:
yes, basically, the response should never be just a message, if the operation was successful the server should reply with information about the created resource so that consumer can refer to it later, so in our case, FE, the consumer, needs to know the id of the unapproved dataset that was created, so that on successful POST, FE can actually open the created resource for the user, if the user has saved the dataset, server replies with datasetId, so thet FE can open page for the user:
databoom.com/dataset/440, so that the user can review their created dataset or whatever, the message string IMO is useless, since the FE should display the error

Reply:
the message about success is useless, because the 201 code tells them that the operation was a success